### PR TITLE
	#165167323 Create a folder for each component

### PR DESCRIPTION
**Pull request by kelvinndmo**
## Description ##
This pull request helps create a folder for each component.Inside the component folder, everything related to the component is stored there.

## Type of change ##
Please select the relevant option
- [ ] Bug fix(a non-breaking change which fixes an issue)
- [ ] New feature(a non-breaking change which adds functionality)
- [ ] Breaking change(fix of feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes
- [ ] End to end test
- [ ] Integration test
- [ ] unit test

## Checklist: ##
- [ ] My code follows the style guidelines of this project
- [x] I have linted my code prior to submission
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes

## Related Pivotal Tracker stories ##
[165167323](https://www.pivotaltracker.com/story/show/165167323)

## Reviewers ##
- @andela/olympian-devs
- @Derrickkip

### CODE REVIEW

**Review by mnswaleh — APPROVED** [review]

**Comment by Derrickkip** [comment]
The title does not follow the engineering guide

**Comment by kelvinndmo** [comment]
> The title does not follow the engineering guide
fixing that

**Comment by kipropbrian** [comment]
Neither does the commit message, this is the third time I am mentioning this to you. Swaleh, how did you approve this PR? Did you look into it carefully?

**Comment by mnswaleh** [comment]
> Neither does the commit message, this is the third time I am mentioning this to you. Swaleh, how did you approve this PR? Did you look into it carefully?

I di not actually check the commit message sorry. I focused on the file changes

**Comment by kelvinndmo** [comment]
> Neither does the commit message, this is the third time I am mentioning this to you. Swaleh, how did you approve this PR? Did you look into it carefully?

just realized its an early while committing where i start with # and vim ignores that first part because i correctly committed using the story id,scope and added a short description plus body and footer

**Review by mnswaleh — APPROVED** [review]

**Review by Kibetchirchir — APPROVED** [review]

File: src/components/Home/Home.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
```
Comment (Kibetchirchir) [inline src/components/Home/Home.js]: Remove unused variables

**Pull request merged**